### PR TITLE
Adding issue guidance page

### PR DESCRIPTION
**Pull request by samedder**

### CODE REVIEW

File: docs/issue_guidance.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+Issues are triaged using GitHub labels, the following labels are used to indicate the status
+of the issue.
+
+## Bug, enhancement and question
+
+Most labels are used according the GitHub standards, these labels indicate the type of issue.
+
+|     Label     |                                             Description                                             |
+| ------------- | --------------------------------------------------------------------------------------------------- |
+| investigating | The issue isn't clear, further investigation is needed to determine if this a bug or something else |
```
[the remark this answers is not in the document]
Comment (samedder) [reply docs/issue_guidance.md]: I'm okay with users assigning an area if we stress that they shouldn't guess, and instead used `needs triage`.

I think the only other thing that we need is like a `fixing` tag? Thoughts @bpm-ms @vturecek ?

File: docs/issue_guidance.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+| question      | The issue is related to a clarification on how Service Fabric works                                 |
+
+
+## Area and needs triage
+
+Area labels and triage labels are used to identify the related involved components of Service Fabric.
+
+|    Label     |                                Description                                |
+| ------------ | ------------------------------------------------------------------------- |
+| needs triage | The involved component isn't clear, further investigation is needed       |
```
Comment (bpm-ms) [inline docs/issue_guidance.md]: do we want to introduce spaces in our tags? Maybe `needs-triage` instead? What do you think about sticking to dashes for multi-word tags? Whatever we choose we should be consistent, and maybe document somewhere on the GitHub the standard we've chosen.

File: docs/issue_guidance.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (bpm-ms) [reply docs/issue_guidance.md]: @anmolah  I want to make sure that we do not actually "Assign" in Github unless someone is actively working on it or plans to in the near future.  This being due to our previous discussion that we want anyone to potentially pick it up, and we don't want issues assigned to someone prematurely who has no capacity to work on them.  

What about a "needs-investigation" label, and @ mention with a stipulation that anyone @ mentioned should follow up and remove the investigation label? I don't really like the "understood" because it implies a full understanding of what's wrong, and doesn't add much value.  I think removing a "needs-investigation" label could serve a similar purpose.  If the tag is removed, and the area expert has posted their initial findings, it brings the issue into a state most ready to be picked up and actually worked on: 
- Initial investigation completed and info posted in the issue
- Not assigned to anyone, ready for someone to pick it up
- Area assigned, with area owners able to track what active issues exist in their area, and with the ability to assign someone from their team, or themselves to something that needs to be worked on.

I don't like a "fixing" tag, this term only works for bugs.  I think "in-progress" might sound better. 

Basically, I think assignment should only be done if that person specifically intends to allocate capacity to fix the issue soon, and once they start working on it, they can add an in-progress tag.

**Comment by athinanthny** [comment]
New issue creation template created to address the ask in this issue. https://github.com/microsoft/service-fabric/blob/master/.github/ISSUE_TEMPLATE
